examples/cpp/BUILDING.md

```diff
@@ -0,0 +1,62 @@
+# Building
+
+## Prerequisites (All Platforms)
+* [CMake 3.5](https://cmake.org/) or newer
+* C++11 compiler *(see platform specific requirements)*
+* [Apache Geode](http://geode.apache.org/releases/) binaries installed or available to link against
+
+   Running requires access to an installation of Geode. By default the value of `GEODE_HOME` is used as the path for the startserver.sh script, if gfsh is not in the path.
+
+### Platform-Specific Prerequisites
+* [Mac OS X](#mac-os-x)
+* [Linux](#linux)
+* [Solaris](#solaris)
+* [Windows](#windows)
+
+## Steps to build
+
+**Mac OS X / \*nix**
+
+    $ cd <example>
+    $ mkdir build
+    $ cd build
+    $ cmake ..
+    $ cmake --build . -- <optional parallelism parameters>
+
+**Windows**
+
+    $ cd <example>
+    $ md build
```

Comment:
`mkdir` exists on windows so this set of instructions does not differ from the one above.